src/test/debuginfo/issue-46457-no-mangle-static.rs

```diff
@@ -0,0 +1,40 @@
+// Copyright 2013-2014 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+// ignore-bitrig
+// ignore-solaris
+// ignore-windows failing on win32 bot
+// ignore-freebsd: gdb package too new
+// ignore-tidy-linelength
+// ignore-lldb
+// ignore-android: FIXME(#10381)
+// compile-flags:-g
+
+// gdb-command: run
+
+// gdb-command: set lang c
+
+// gdb-command: print NO_MANGLE_DEADBEEF
+// gdb-check:$1 = 3735928559
+
+// gdb-command: ptype NO_MANGLE_DEADBEEF
+// gdb-check:$2 = u6
```

Comment:
This should probably say `u64`

Reply:
Ha yup sorry was trying to get tests to fail on my machine as per above comment, will fix

Reply:
@m4b it did [fail in travis](https://travis-ci.org/rust-lang/rust/builds/315985665#L1875), it seems

Reply:
Yes but it doesn’t on my local machine, using above instructions. I do not know why...